test/common.py

```diff
@@ -605,6 +605,7 @@ def setUp(self):
           # Opt in to node v15 default behaviour:
           # https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode
           self.node_args.append('--unhandled-rejections=throw')
+      self.emcc_args += building.get_emcc_node_flags(node_version)
```

Comment:
This will mean that we are passing `-sMIN_NODE_VERSION` to pretty much every single test.

How about if `get_emcc_node_flags` returns nothing when the current node version is >= default MIN_NODE_VERSION?     i.e. it should only be needed when the configured version of node is older that the default min I think?

Reply:
Sounds good, done.